samples/cpp/windows/console/samples/intent_recognition_samples.cpp

```diff
@@ -222,3 +222,86 @@ void IntentContinuousRecognitionWithFile()
     recognizer->StopContinuousRecognitionAsync().get();
     // </IntentContinuousRecognitionWithFile>
 }
+
+// Intent recognition using pattern matching and microphone.
+void IntentRecognitionWithPatternMatchingAndMicrophone()
+{
+    // <IntentRecognitionWithPatternMatchingAndMicrophone>
```

Comment:
>IntentRecognitionWithPatternMatchingAndMicrophone [](http://example.com/codeflow?start=8&length=49)

Is this a new code tag?
If so you need to update the white list...

Reply:
you will notice when you build this in carbon repo.

Reply:
will do